packages/components/src/ChainNetwork.ts

```diff
@@ -652,8 +829,66 @@ export const polygonMumbaiRPC = [
 ] as string[];
```

Comment:
type annotation over type cast

Reply:
yeah we have alot of type cast, so could be a seperate feature issue to work on those